packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/knowledge/KnowledgeController.java

```diff
@@ -122,6 +122,48 @@ void init() {
 		taskService.addResponseHandler(equationsCleanupResponseHandler);
 	}
 
+	/**
+	 * Clean up a list of equations
+	 *
+	 * @return List of cleaned-up equations
+	 */
+	@PostMapping("/get-cleaned-equations")
+	@Secured(Roles.USER)
```

Comment:
Would prefer better wording here, you are POST'ing to a "GET" which might be a little confusing. Maybe better just to have `POST /clean-equations`.